testv2/testv2_auth/test_smart_oauth.py

```
@@ -0,0 +1,360 @@
+"""
+Tests for the SmartOAuth client that verifies it works correctly in both sync and async contexts.
+"""
+
+import pytest
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)